examples/defining-classes/defining_classes.bal

```diff
@@ -1,10 +1,10 @@
 import ballerina/io;
 
 public class Counter {
-    // `private` means accessible only by code within the class definition.
+    // `private` means accessible only by code within the `class` definition.
     private int n;
 
-    // `init` method initializes the object.
+    // `init` method initializes the `object`.
```

Comment:
```suggestion
    // `init` method initializes the object.
```
Slightly misleading to format "object" as code here since here it's referring to an object value created using this class definition. When formatted as code, `object` would come off as the keyword and by its usage, the object type descriptor.